Sources/QuickObjectiveC/QuickSpec.m

```diff
@@ -84,13 +84,12 @@ + (void)buildExamplesIfNeeded {
     [QuickConfiguration class];
     World *world = [World sharedWorld];
 
-    ExampleGroup *rootExampleGroup = [world rootExampleGroupForSpecClass:self];
-
-    if ([rootExampleGroup examples].count > 0) {
+    if ([world isRootExampleGroupInitializedForSpecClass:[self class]]) {
         // The examples fot this subclass have been already built. Skipping.
         return;
     }
 
+    ExampleGroup* rootExampleGroup = [world rootExampleGroupForSpecClass:[self class]];
```

Comment:
`ExampleGroup* rootExampleGroup` should be `ExampleGroup *rootExampleGroup` for consistency.

Reply:
Sorry, old habit. Fixed.